scrypto/src/runtime/runtime.rs

```diff
@@ -75,6 +75,11 @@ impl Runtime {
         ScryptoEnv.get_transaction_hash().unwrap()
     }
 
+    /// Returns the transaction hash.
+    pub fn generate_ruid() -> [u8; 32] {
+        ScryptoEnv.generate_ruid().unwrap()
```

Comment:
lol did we never remove the actual scrypto env call?

Reply:
Yeah, possibly due to some native things depending on it.